doc/api/stream.md

```diff
@@ -489,6 +489,16 @@ added: v11.4.0
 
 Is `true` if it is safe to call [`writable.write()`][stream-write].
 
+##### writable.writableCorked
+<!-- YAML
+added: REPLACEME
+-->
+
+* {Integer}
+
+Number of times [`writable.uncork()`][stream-uncork] needs to be
```

Comment:
Missing bottom reference.

Reply:
@vsemozhetbyt does it need one? Isn't [stream-uncork] enough?